focal/4_gs1_identification.html

```diff
@@ -0,0 +1,336 @@
+            or the other occurs.
+      </p>
+      <h5>Replacement</h5>
+      <p>
+            Replacement is similar to revocation in that it invalidates the credential, but it indicates that there is
+            another, equivalent credential available. The most common use case for this is in acquisitions and mergers, as
+            defined in the GS1 General Specifications:
+      </p>
+      <p>
+            <em>
+                  During an acquisition or merger, a company may assume responsibility for the acquired company's GS1 Company
+                  Prefix and/or individual GS1 identification key licences. In the situations where the licences transfer, the
+                  acquiring company can:
+            </em>
+      </p>
+      <ul>
+            <li>
+                  <em>
+                        Use the acquired company's GS1 Company Prefix(es) and GS1 identification key(s
+                  </em>
+            </li>
+            <li>
+                  <em>
+                        Issue GS1 identification keys using the newly acquired GS1 Company Prefix(es)
+                  </em>
+            </li>
+      </ul>
+      <p>
+            <em>
+                  For example, products that the acquired company identified using its GS1 Company Prefix or individual GS1
+                  identification key licences can still be produced using the same GTINs after the merger. Additionally,
+                  parties, locations, assets, and other objects identified with GS1 identification keys can continue to use
+                  those keys after the merger.
+            </em>
+      </p>
+      <p>
+            <em>
+                  If a partial purchase occurs, where only a segment of a larger entity is acquired, the involved companies must
+                  determine whether GS1 identification licences are transferred based on their specific business
+                  requirements.
+            </em>
+      </p>
+      <p>
+            In such a situation, the acquiring company takes over the licenses of the acquired company and should be issued
+            the appropriate credentials. Those originally issued to the acquired company are no longer valid, but simple
+            revocation could be highly disruptive as there may be thousands of extension credentials that could be invalidated
+            by the business rules that apply to revocation. Instead, the replacement status indicates that the licence
+            credential has been replaced. As with revocation, any new extension credentials that directly reference the
+            replaced credential are invalid, but pre-existing extension credentials should be validated against the
+            replacement credential using the normal business rules.
+      </p>
+      <p>
+            Suppose that Healthy Tots is acquired by Benevolent Conglomerate. Benevolent Conglomerate may decide on a
```

Comment:
Please add Benevolent Conglomerate to the actors section.

Reply:
Added.